### DISCUSSION TITLE
chunkById in descending order

### DISCUSSION THREAD

**Discussion by tuimz** [laravel/framework#40780] [Ideas]
Hi,

Just checking if there would be interest in this. 

I have a specific use case for this and want to check if this is something that would be merged before I start writing code / tests and make a PR.

**Comment by gabrielpeixoto** [comment]
You can use lazyByIdDesc as follows:
```php
Model::lazyByIdDesc(5000)->each(function ($item) {
    // use $item
});
```

This will loop through using the following queries:
```sql
select * from `model` order by `id` desc limit 5000
select * from `model` where `id` < 20000 order by `id` desc limit 5000;
select * from `model` where `id` < 15000 order by `id` desc limit 5000;
...

```
